### PR TITLE
feat(ui): Redesigned release health index page

### PR DESCRIPTION
**Pull request by matejminar**
New iteration of release health index page that supports per project releases and ordering.

Screenshot 2020-02-27 at 18 04 43

Couple of minor things still different than in figma.
This is just a playground for us, behind `releases-v2` feature flag.

### CODE REVIEW

File: src/sentry/static/sentry/app/views/releasesV2/list/index.tsx [hunk 1]
```diff
@@ -46,28 +49,79 @@ class ReleasesList extends AsyncView<Props, State> {
...
       query: {...location.query, query},
     });
   };
 
+  handleSort = (sort: string) => {
+    const {location, router} = this.props;
+
+    router.push({
+      ...location,
+      query: {...location.query, sort},
```
Comment (billyvg) [inline src/sentry/static/sentry/app/views/releasesV2/list/index.tsx]: Should sorting reset cursor?

File: src/sentry/static/sentry/app/views/releasesV2/list/index.tsx [hunk 2]
```diff
@@ -135,5 +189,21 @@ class ReleasesList extends AsyncView<Props, State> {
...
+  margin-bottom: 0;
+  ${PageHeading} {
+    margin-bottom: ${space(2)};
+  }
+`;
+const SortAndFilterWrapper = styled('div')`
+  display: flex;
+  margin-bottom: ${space(2)};
+
+  & > *:not(:last-child) {
```
Comment (billyvg) [inline src/sentry/static/sentry/app/views/releasesV2/list/index.tsx]: What about using css grid instead of this complex selector

**Review by billyvg — APPROVED** [review]

File: src/sentry/static/sentry/app/views/releasesV2/list/index.tsx (hunk 1 above)
Comment (matejminar) [reply src/sentry/static/sentry/app/views/releasesV2/list/index.tsx]: yeah, let's reset 👍

File: src/sentry/static/sentry/app/views/releasesV2/list/index.tsx (hunk 2 above)
Comment (matejminar) [reply src/sentry/static/sentry/app/views/releasesV2/list/index.tsx]: ❤️

**Pull request merged**